components/Microcopy/Microcopy.tsx

```diff
@@ -0,0 +1,60 @@
+import { FunctionComponent } from "react";
+import { TypeMicrocopy } from "../../generated/contentful";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTriangleExclamation } from "@fortawesome/free-solid-svg-icons";
+import Style from "./styles.module.scss";
+
+export interface MicrocopyProps {
+  entries: TypeMicrocopy[];
+  id: string;
+}
+
+const Microcopy: FunctionComponent<MicrocopyProps> = ({ entries, id }) => {
+  let microcopy = entries.find((entry) => entry.sys.id == id);
+
+  if (
+    typeof window !== "undefined" &&
+    window.sessionStorage.getItem("editingMode")
+  ) {
+    // client side
+    if (microcopy) {
+      if (microcopy) {
+        return (
+          <span
+            className={Style.editing}
+            onClick={() => {
+              window.open(
+                `https://app.contentful.com/spaces/dtb5w0ega2aw/entries/${id}`,
+                "_blank"
+              );
+            }}
+          >
+            {microcopy.fields.content}
```

Comment:
instead of repeating the whole if return, you should just do it on this line. `microcopy ? microcopy.fields.content : <fontawesome>`